CPS-XXXX/README.md

```diff
@@ -0,0 +1,134 @@
+These two specifications do not address an observed growth rate less than 2160 per 36 hr but greater than zero, nor does any other specification.
+
+Such low growth rates become plausible when the assumptions of the Praos argument are violated for an extended period of time.
+Global-scale disasters, infrastructure attacks, local outages, eclipse attacks, bugs, negligent misconfiguration, etc could all cause it.
+
+## Problem
+
+The following mutually exclusive options seem to cover the design space, but each has unique advantages and disadvantages.
+
+- *AssumeImmutableAge* (AssumeIA).
+  The node will not switch to a chain if doing so would rollback more than 2160 blocks regardless of their age.
+  This limit is justified by the Common Prefix property of the Praos security argument as instantiated for Cardano.
+  (Note the absence of the Chain Growth property.)
+
+- *EnforceImmutableAge* (EnforceIA).
+  As a refinement of AssumeIA, the node will not switch to a chain if doing so would rollback more than 2160 blocks and/or a block that is more than 36 hr old (TODO: should age be with respect to the wall clock or the selection's tip?).
+  This limit is a disjunction of Common Prefix and Chain Growth.
+
+- *EnforceChainGrowth* (EnforceCG).
+  In addition to AssumeIA, the node rejects any chain that has less than 2160 blocks per 36 hr (even if selecting it wouldn't require rolling back any blocks).
+
+These options can be compared by considering the following scenarios.
+
+- *TwoThirdsInaccessible*.
+  Suppose more than 2/3rds of stake was knocked offline (eg by a solar flare) such that less than 1/3rd of all stake was active for more than 36 hr, but it all remained connected.
+
+    - The other scenarios all involve this one to some degree, so it serves as a baseline.
+
+    - If the adversary controls less than half of the surviving stake, then it still cannot defeat Praos during the outage.
+
+    - Note that the chain would have a growth rate less than 2160 per 36 hr but greater than zero.
+      
+    - If the whole network implemented AssumeIA, then the 2161th youngest block would be at least three times older than usual (36+ hr instead of 12 hr).
+      Since the nodes and/or other Cardano community tooling might have been designed around the guarantee of at most 36 hrs for 2160 blocks, they could fail outright or become more vulnerable to possible DoS attacks during the outage.
+
+    - If the whole network implemented EnforceIA, then the nodes would not face increased risks.
+      The 2160 disjunct in the EnforceIA definition is an optimization for when the chain is growing faster than 2160 blocks per 36 hr.
+
+    - If the whole network implemented EnforceCG, then the Recovery Plan would be unavoidable (once the affected stake was back online).
+      The silver lining is that all nodes would be able to switch to the repaired chain automatically, without needing manual intervention.
```

Comment:
Likewise, I would elaborate more on this case - would grow the chain up until the 36 hour point, when all nodes would stop growing the chain and we would need to effect disaster recovery. But since we hadn't grown more than 2160 blocks, each node would be happy to roll back and select the denser chain...

Reply:
I added a bit here. And a TODO... I haven't yet done the math to bound how many blocks might slowly arise before the rule actually kicked in.